Spring.FluentContext.Web/Spring.FluentContext.Web.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please align this project content to the `Spring.FluentContext`, covering symbols, nuget definitions etc.

Reply:
Aligned